default.yml

```diff
@@ -78,15 +68,32 @@ Metrics/ParameterLists:
 Metrics/BlockLength:
   Enabled: false
 
-Style/DoubleNegation:
-  Enabled: false
-
 Naming/FileName:
   Exclude: ['Gemfile', 'Guardfile']
 
 Naming/UncommunicativeMethodParamName:
   Enabled: false
 
+Style/AccessModifierDeclarations:
```

Comment:
This is a new rule that has a ton of violations in API

Reply:
I think we should switch this to `EnforcedStyle: inline` and embrace the API style, it's the most used throughout our codebases and makes it explicit on each method if it's private. Sound ok to you?